source/Cosmos.IL2CPU/IL/Ldlen.cs

```diff
@@ -1,25 +1,31 @@
+using Cosmos.IL2CPU.ILOpCodes;
 using XSharp;
 using XSharp.Assembler;
 using static XSharp.XSRegisters;
 
 namespace Cosmos.IL2CPU.X86.IL
 {
-    [OpCode(ILOpCode.Code.Ldlen)]
-    public class Ldlen : ILOp
-    {
-        public Ldlen(Assembler aAsmblr)
-            : base(aAsmblr)
-        {
-        }
+            [OpCode(ILOpCode.Code.Ldlen)]
+            public class Ldlen : ILOp
+            {
+                public Ldlen(Assembler aAsmblr)
+                    : base(aAsmblr)
+                {
+                }
 
-        public override void Execute(_MethodInfo aMethod, ILOpCode aOpCode)
-        {
-            DoNullReferenceCheck(Assembler, DebugEnabled, 4);
+                public override void Execute(_MethodInfo aMethod, ILOpCode aOpCode)
+                {
+                    Assemble(Assembler, DebugEnabled);
```

Comment:
What are the changes here intended to do? No reason to add an extra method call.